index.js

```diff
@@ -0,0 +1,26 @@
+// import express (after npm install express)
+const express = require('express');
+
+// create new express app and save it as "app"
+const app = express();
+
+// server configuration
+const PORT = 3000;
+
+// JSON Data
+var data  = require('./package.json');
```

Comment:
better to rename this variable as `packageData`, Because it should be specific among the file.

Reply:
Thanks, Done!